pylabrobot/config/config.py

```diff
@@ -0,0 +1,53 @@
+import logging
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import Union
+
+LOG_FROM_STRING = {
+  "DEBUG": logging.DEBUG,
+  "INFO": logging.INFO,
+  "WARNING": logging.WARNING,
+  "ERROR": logging.ERROR,
+  "CRITICAL": logging.CRITICAL,
+}
+
+LOG_TO_STRING = {
+  logging.DEBUG: "DEBUG",
+  logging.INFO: "INFO",
+  logging.WARNING: "WARNING",
+  logging.ERROR: "ERROR",
+  logging.CRITICAL: "CRITICAL",
+}
+
+
+@dataclass
+class Config:
+  """The configuration object for the application."""
+
+  @dataclass
+  class Logging:
+    """The logging configuration."""
+    level: int = logging.INFO
+    log_dir: Union[str, Path] = Path(".")
```

Comment:
would it make things easier to just convert to a Path before storing in Config?

Reply:
Yes, good call. Taking the union out!